charts/node-server/templates/deployment.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     metadata:
       labels:
         app: tensorleap-node-server
-        revision: {{ .Values.image_tag }}
+        revision: { { .Values.image_tag } }
```

Comment:
Your linter broke this (happens to me as well sometimes)

Reply:
I think my linter doesn’t work on yaml files.
I will search to fix it, thanks!